tools/run_tests/xds_k8s_test_driver/framework/rpc/grpc_testing.py

```diff
@@ -39,6 +39,8 @@
 MethodStats = messages_pb2.LoadBalancerAccumulatedStatsResponse.MethodStats
 RpcsByPeer = messages_pb2.LoadBalancerStatsResponse.RpcsByPeer
 
+_HOOK_SERVER_PORT = 8000
```

Comment:
Is this hardcoded in the implementation as well?

Reply:
Hardcoded in the `grpcurl` invocation in the server yaml file. I'm working with Eugene on an improved hook server API that doesn't require port management at all.

Reply:
Conversation moved to https://github.com/grpc/psm-interop/pull/34#discussion_r1484908031.